dbcon/mysql/ha_mcs_execplan.cpp

```diff
@@ -5914,7 +5930,10 @@ void gp_walk(const Item* item, void* arg)
 
         // bug 3137. If filter constant like 1=0, put it to ptWorkStack
         // MariaDB bug 750. Breaks if compare is an argument to a function.
-        //				if ((int32_t)gwip->rcWorkStack.size() <=  (gwip->rcBookMarkStack.empty() ? 0
+        //				if ((int32_t)gwip->rcWorkStack.size() <=
+        //(gwip->rcBookMarkStack.empty()
+        //?
+        // 0
         //: gwip->rcBookMarkStack.top())
         //				&& isPredicateFunction(ifp, gwip))
```

Comment:
The above comment does not look readable, can you please fix this?

Reply:
JFYI I am following the convention we all accepted previously, namely to use auto-formater. I am using clang-format in this case so this is an automated change. IMHO if the accepted tool formats the code this way it is OK with me.